packages/shared/src/lib/auth.ts

```diff
@@ -42,6 +42,7 @@ export enum AuthTriggers {
   RanksModal = 'ranks modal',
   Onboarding = 'onboarding',
   SubmitNewSource = 'submit new source',
+  Downvote = 'downvote',
```

Comment:
Just make sure to also log this in the Analytics google sheet for reference to Sab.